assets/stylesheets/base/_global.scss

```diff
@@ -2,7 +2,28 @@
   font-size: $base-font-size;
 }
 
+:focus {
+  outline: 0;
+}
+
 body {
   font-size: $base-font-size;
   @extend .font-family-base;
+}
+
+%clearfix {
```

Comment:
This mixin should already be defined by Bootstrap. 

https://github.com/twbs/bootstrap-sass/blob/master/assets/stylesheets/bootstrap/mixins/_clearfix.scss